models/sessions/scratch/snowflake/snowplow_web_sessions_this_run.sql

```diff
@@ -231,6 +254,13 @@ select
     a.geo_latitude,
     a.geo_longitude,
     a.geo_timezone,
+    b.last_geo_country,
+    b.last_geo_region_name,
+    b.last_geo_city,
+    a.country,
+    a.continent,
+    b.last_country,
+    b.last_continent,
```

Comment:
Could you order these so they are all the firsts then all the lasts, I would probably also add the `geo_` prefix to the country and continent fields so they are consistent. Same across bigquery.